DigitalLearningSolutions.Web/Views/TrackingSystem/CourseSetup/CourseContent/EditSection.cshtml

```diff
@@ -44,7 +44,7 @@
       <input type="hidden" asp-for="SectionName" />
 
       <div class="nhsuk-grid-row  nhsuk-u-margin-bottom-4">
-        <partial name="_EditSectionDiagnostic" model="Model"/>
+        @*<partial name="_EditSectionDiagnostic" model="Model"/>*@

```

Comment:
Was leaving this commented out intentional?

Reply:
Ah whoops no